exchanges/okcoin/okcoin_wrapper.go

```diff
@@ -158,3 +160,47 @@ func (o *OKCoin) UpdateTradablePairs(forceUpdate bool) error {
 	return o.UpdatePairs(currency.NewPairsFromStrings(pairs),
 		asset.Spot, false, forceUpdate)
 }
+
+// UpdateTicker updates and returns the ticker for a currency pair
+func (o *OKCoin) UpdateTicker(p currency.Pair, assetType asset.Item) (tickerData ticker.Price, _ error) {
+	if assetType == asset.Spot {
+		resp, err := o.GetSpotAllTokenPairsInformation()
+		if err != nil {
+			return
```

Comment:
might need to explicitly return error here as its a blank identifier in the return type.